src/dev/build/tasks/create_static_fs_with_node_modules_task.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import del from 'del';
+import globby from 'globby';
+import normalizePath from 'normalize-path';
+import { resolve } from 'path';
+import { generateStaticFsVolume } from 'static-fs';
+
+async function deletePathsList(list) {
+  for (const path of list) {
+    await del(path);
+  }
+}
+
+async function getTopLevelNodeModulesFolders(rootDir) {
+  const nodeModulesFoldersForCwd = await globby(
+    ['**/node_modules', '!**/node_modules/**/node_modules'],
+    {
+      cwd: rootDir,
+      onlyDirectories: true,
+    }
+  );
+
+  return nodeModulesFoldersForCwd.map(folder => normalizePath(resolve(rootDir, folder)));
```

Comment:
`normalizePath()` forces the use of unix slashes on windows, why would we want that? We needed to [do the opposite here](https://github.com/elastic/kibana/blob/master/packages/kbn-optimizer/src/optimizer/kibana_platform_plugins.ts#L49) because the general expectation is that absolute paths should use the system separators, not weird normalized paths like `c:/x/y/x` (`c:\x\y\x` being the correct path)

Reply:
in this specific case you're right, we don't need it, but normalizePath is very useful